coding/compressed_bit_vector.cpp

```diff
@@ -312,6 +312,28 @@ unique_ptr<CompressedBitVector> CompressedBitVectorBuilder::FromBitGroups(
   return make_unique<SparseCBV>(setBits);
 }
 
+// static
+unique_ptr<CompressedBitVector> CompressedBitVectorBuilder::FromCBV(CompressedBitVector const & cbv)
+{
+  auto strat = cbv.GetStorageStrategy();
+  switch (strat)
+  {
+  case CompressedBitVector::StorageStrategy::Dense:
+  {
+    DenseCBV const & dense = static_cast<DenseCBV const &>(cbv);
+    auto bitGroups = dense.m_bitGroups;
+    return CompressedBitVectorBuilder::FromBitGroups(move(bitGroups));
```

Comment:
It is tricky: we already have
```FromBitGroups(vector && bitGroups)```
which pop_back's empty groups so ```const``` has to be dropped.

I would refrain from implementing the ```const &``` version until we need it and we do not now because FromCBV() has been replaced with a call to Clone().